### ISSUE TITLE
Separate the auto commits from our legit commits

### ISSUE THREAD

**Issue by scoskey** [mathblogging/mathblogging.org#18]
Maybe we should make the auto commits come from a fake user?

**Comment by pkra** [comment]
I'm wondering why that's the case anyway (i.e., why are the commits linked to your account?) but yes, that makes sense.

**Comment by scoskey** [comment]
Maybe because I cloned it originally?

**Comment by pkra** [comment]
Indeed, that would be way more elegant... _bows his head in amazement towards clear superiority_

**Comment by pkra** [comment]
Is this still relevant?
